src/SFA.DAS.PAS.Account.Api/ApplicationInsights.config

```diff
@@ -82,7 +72,7 @@
         
         NOTE: handler configuration will be lost upon NuGet upgrade.
         -->
-				<Add>System.Web.Handlers.TransferRequestHandler</Add>
+        <Add>System.Web.Handlers.TransferRequestHandler</Add>
```

Comment:
The App Insights nuget upgrade removed this line and when I saw 'Transfers' I assumed it was levy transfers, rather than server transfers, so I put it back.
Of course, it refers to server transfers, so it makes sense to go with the original nuget removal.
Removed.